server/php-queries-types.cpp

```diff
@@ -0,0 +1,197 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2021 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <poll.h>
+
+#include "common/kprintf.h"
+#include "server/php-queries-types.h"
+#include "server/php-worker.h"
+#include "server/php-runner.h"
+#include "server/php-engine.h"
+#include "server/php-mc-connections.h"
+#include "server/php-sql-connections.h"
+
+void php_query_x2_t::run(php_worker *worker __attribute__((unused))) noexcept {
+  query_stats.desc = "PHPQX2";
+
+  php_script_query_readed(php_script);
+
+  //  worker->conn->status = conn_wait_net;
+  //  worker->conn->pending_queries = 1;
+
+  static php_query_x2_answer_t res;
```

Comment:
Why static?

Reply:
Because pointer to `res` is stored to `ans` below and later used after returning from this function and swapping context back